templates/company-board/test/company-board.js

```diff
@@ -183,13 +183,13 @@ contract('Company with board', ([_, owner, boardMember1, boardMember2, shareHold
         shareTokenManager = TokenManager.at(installedApps['token-manager'][1])
       })
 
-      it('costs ~10.4e6 gas', async () => {
+      it('costs max ~10.29e6 gas', async () => {
```

Comment:
Mmm admittedly this is less of an optimization than I had anticipated.

Reply:
yep, it saves only one call

Reply:
I don't think that it's minor at all. It's about 5k gas per app. The measurements were too coarse.